src/plugins/expressions/public/interpreter_provider.ts

```diff
@@ -78,7 +82,7 @@ export function interpreterProvider(config: InterpreterConfig): ExpressionInterp
     const link = chain.shift(); // Every thing in the chain will always be a function right?
     if (!link) throw Error('Function chain is empty.');
     const { function: fnName, arguments: fnArgs } = link;
-    const fnDef = getByAlias(functions, fnName);
+    const fnDef = getByAlias(functions.toJS(), fnName);
```

Comment:
Because interpreter provider now receives a function registry instead a `{}` copy. But `getByAlias` function needs a `{}` map of all functions and `.toJS()` creates the map.